liquidprompt

```diff
@@ -633,7 +633,7 @@ __lp_source_config() {
         done
     fi
 
-    # Delete this code in version 1.11
+    # FIXME Delete this code in version 1.11
```

Comment:
I'd rather just remove this completely, since we are just keeping these shims to prevent breakage.